### ISSUE TITLE
find-motifs

### ISSUE THREAD

**Issue by jpinus** [MicrobialDarkMatter/nanomotif#33]
I use nanomotif 0.2.1 as an conda environment and followed the instructions outlined in the documentation(https://nanomotif.readthedocs.io/en/latest/required_files.html) meticulously for the input files. However, I encountered a couple of issues with the initial command execution of nanomotif.

The command mentioned in the documentation nanomotif find_motifs ASSEMBLY.fasta PILEUP.bed CONTIG_BIN.tsv -t THREADS --out OUT is not running as expected. It seems there's a discrepancy in the command syntax. According to the error message I received, "find_motifs" should be "find-motifs".
Even after correcting the command syntax to "nanomotif find-motifs assembly.fasta OUT.bed contig_bin.tsv -t 60 --out OUT_nanomotif", I encountered another issue. The execution didn't produce the expected output file "bin-motifs.tsv". It appears that this might be due to the omission of "CONTIG_BIN.tsv" in the initial command.

**Comment by jpinus** [comment]
@SorenHeidelbach I use the conda installation:
command: conda install -c bioconda nanomotif

this is installing nanomotif 0.1.20

How can I specify that the newest version, 0.4.1 can be installed?

**Comment by SorenHeidelbach** [comment]
Yes, the bioconda channel is currently facing some issues regarding checks. We will upload the newest version once it has been fixed. 

Currently you can intall the newest version using pip. 
![image](https://github.com/MicrobialDarkMatter/nanomotif/assets/42008781/b5662ca1-01a8-40ab-b5c5-c25230f5c0d1)
